hw04_struct_comparator/main.go

```diff
@@ -1,5 +1,121 @@
 package main
 
+import "fmt"
+
+type Book struct {
+	id     int
+	title  string
+	author string
+	year   int
+	size   int
+	rate   float32
+}
+
+// enum компараторов
+
+type Comparator int
```

Comment:
идеоматичне называть с таким же префиксом, котрый будет и у констант

```suggestion
type Compare int

const (
	CompareByYear Comparator = iota
	CompareBySize
	CompareByRate
)
```